File: Domain/Entities/DB/Status.cs
using System;

namespace Domain.Entities.DB;
public class Status : EntityBase
{
    public Guid? Id { get; set; }
    public string TableName { get; set; }
    public string ColumnName { get; set; }
    public string Code { get; set; }
    public string DescTh { get; set; }
    public string DescEn { get; set; }
    public int Seq { get; set; }
    public string Remark { get; set; }
    public bool? Active { get; set; }
    public string BackgroundColor { get; set; }
    public string FontColor { get; set; }
}

File: Web/ClientApp/src/app/models/db/status.ts
import { Guid } from "guid-typescript";
import { EntityBase } from "../entityBase";

export class Status extends EntityBase {
    id: Guid;
    tableName: string;
    columnName: string;
    code: string;
    descTh: string;
    descEn: string;
    seq: number;
    remark: string;
    active: boolean;
    backgroundColor: string;
    fontColor: string;
}
